README.md

```diff
@@ -1,181 +1,146 @@
+1. Log in to [Mondoo Platform](https://console.mondoo.com)
+2. Select the Space you want to integrate with your repository.
+3. Click on **Settings** and then **Service Accounts**.
+4. Click **ADD ACCOUNT**.
+5. Select the **Base64-encoded** checkbox, and then click on the **GENERATE NEW CREDENTIALS** button.
+6. Copy the base64 encoded credentials and then move on to the next section.
 
-Check the Base64-encoded checkbox, click Generate New Credentials, and copy the generated credentials:
-![Generate Credentials](/assets/credentials.png)
+### Add new GitHub Actions Secrets 
 
-In your GitHub repo, go to Settings -> Secrets -> Actions and click New repository secret. Create a new secret named "MONDOO_CLIENT_ACCOUNT" with the contents you copied from the Mondoo Console:
-![Generate Credentials](/assets/secret.png)
-
-## Workflow Configuration Options
-
-The Mondoo GitHub Action has three required `with` values that must be set in your workflow configuration file:
-
-**service_account_credentials**
-
-The Mondoo service account credentials. Store these in a GitHub secret. Do not set them directly in the workflow configuration file. Once a secret is set up (as shown above) you can reference that secret in your workflow configuration file as `${{ secrets.MONDOO_CLIENT_ACCOUNT }}`.
-
-**scan_type**
-
-The type of Mondoo scan to perform:
-
-- `k8s` for Kubernetes manifest scanning.
-- `terraform` for Terraform configuration file scanning.
-- `docker_image` for scanning of Docker images from a Docker registry or from earlier GitHub actions steps.
-- `docker_image_from_dockerfile` for scanning of Docker images from a Dockerfile. Note: This will build and then scan the image which make be a lengthy process.
-
-**path**
-
-The file to scan with Mondoo or, if `scan_type` is set to `docker_image_from_dockerfile`, the path to the Dockerfile. Examples: `nginx.yml` or `Dockerfile`
-
-**docker_image_name**
-
-The container image name to scan when `scan_type` is set to `docker_image`. ex: `nginx:22.04`
-
-**score_threshold**
-
-The score threshold for scans. Value can be any number from `0-100`. If any score falls below the threshold, exit 1. Default value is `0`.
-
-**output_format**
-
-Set the output format (`compact`|`full`|`csv`|`json`|`junit`|`yaml`). Default `compact`.
-
-**extra_args**
-
-Allows specify extra command-line arguments for the Mondoo scan command.
+1. Click on **Settings** in your GitHub repository.
+2. Under the **Security** section click on **Actions**.
+3. Click **New repository secret**.
+4. Name the secret `MONDOO_SERVICE_ACCOUNT` and paste the base64 encoded credentials from the previous section into the value input.
+5. Click **Add secret**.
 
 ## Examples Workflows
 
 Simple scan of nginx.yml Kubernetes manifest:
 
 ```yaml
-name: mondoo-scan
-
+name: Mondoo Kubernetes Manifest scan
 on:
-  pull_request:
   push:
-    branches: [main]
-
+    paths:
+    - 'k8s/*.yaml'
 jobs:
-  build:
+  install:
```

Comment:
We should give this a more friendly name for what it's doing since it shows up in the UI now